node-base/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#
+# --- Stage 1: Base ---
+#
+
+ARG ALPINE_VERSION="3.22"
+FROM common:${ALPINE_VERSION} AS base
+
+# Install packages
+# hadolint ignore=DL3018
+RUN apk add --no-cache \
+    nodejs \
+    npm
```

Comment:
I think we can remove the `npm` package, we never use it at runtime, we always directly invoke the Node scripts and never install any packages.